recipe/meta.yaml

```diff
@@ -28,10 +27,13 @@ requirements:
     - mock  # [py26 or py27]
```

Comment:
The issue is we have a selector, but selectors don't work with `noarch`. Hence the need to remove `noarch`. We are working on getting something into the linter to catch this sort of thing quicker.